static/js/giphy.js

```diff
@@ -33,12 +34,34 @@ export function focus_current_edit_message() {
 const APPROX_HEIGHT = 350;
 const APPROX_WIDTH = 300;
 
+export function update_giphy_rating() {
+    if (page_params.realm_giphy_rating === 0) {
+        $("#compose_box_giphy_grid").hide();
+    } else {
+        $("#compose_box_giphy_grid").show();
+    }
+}
```

Comment:
Sigh, it appears `#compose_box_giphy_grid` is a duplicate ID -- we need to change it to a class.

(Also, is that the right name for a button?  Maybe `.compose_box_giphy_link` would be more appropriate)